examples/compose-dropdown-button/manifest.json

```diff
@@ -14,7 +14,7 @@
   "web_accessible_resources": [
     {
       "matches": ["https://mail.google.com/*"],
-      "resources": ["pageWorld.js", "monkey.png"]
+      "resources": ["pageWorld.js", "monkey.png", "*.map"]
```

Comment:
Borrows this trick from https://github.com/InboxSDK/hello-world/commit/83f1168fb5d08f056cb7d7a86c4ec532bf2dacf3#diff-c0eb04b279c83eb3e6113c1ec2beeac82c6526bc38ba5287f1a59d236204ba14R26